src/containers/hero/hero.tsx

```diff
@@ -1,15 +1,39 @@
-import Button from '@ui/button';
+// * Import React Hooks
+import { useEffect } from 'react';
+// * Import Types
 import { ButtonContent, HomeSection, Item as ItemType } from '@types';
+// * Import UI Components
+import ClipLoader from 'react-spinners/ClipLoader';
 import Item from '@components/item';
+import Button from '@ui/button';
+// * Import Redux Components
+import useAppDispatch from '../../store/hooks/useAppDispatch';
 import useAppSelector from '../../store/hooks/useAppSelector';
-import { selectNewest } from '../../features/leda-nft/store/leda-nft.slice';
+import { getNewest } from '../../features/marketplace/store/marketplace.actions';
+import { selectMarketplaceState } from '../../features/marketplace/store/marketplace.slice';
 
 type Props = {
   homeSection?: HomeSection;
 };
 
+const LoadingSpinner = () => (
+  <div className="text-center">
+    <ClipLoader className="spinner" color="#35b049" />
+  </div>
+);
+
+// ? Render Hero Component
```

Comment:
same with this comment and rest of them. I'd suggest to not include anything like this.